www/src/components/homepage/homepage-newsletter.js

```diff
@@ -11,100 +12,90 @@ import { mediaQueries } from "gatsby-design-tokens/dist/theme-gatsbyjs-org"
 
 const stripedBorderHeight = 1
 
-const Container = styled(`div`)`
-  background: ${p => p.theme.colors.newsletter.background};
-  border: 1px solid ${p => p.theme.colors.newsletter.border};
-  border-radius: ${p => p.theme.radii[2]};
-  display: flex;
-  flex-direction: column;
-  margin-bottom: ${p => p.theme.space[8]};
-  padding: calc(${p => p.theme.space[8]} * 1.2);
-  padding-bottom: calc(
-    ${props => rhythm(props.theme.space[8] * 1.2)} +
-      ${p => p.theme.space[stripedBorderHeight]}
-  );
-  position: relative;
+const containerStyles = {
+  bg: `newsletter.background`,
+  border: t => `1px solid ${t.colors.newsletter.border}`,
+  borderRadius: 2,
+  flexDirection: `column`,
+  mb: 8,
+  p: t => `calc(${t.space[8]} * 1.2)`,
```

Comment:
Isn't this just space[9]?

40 * 1.2 = 48

Reply:
LOL yes :)

Reply:
🙏